python/lsst/pipe/supertask/cmdLineFwk.py

```diff
@@ -187,14 +205,25 @@ def parseAndRun(self, argv=None):
             graph2dot(qgraph, args.qgraph_dot)
 
         # optionally dump some info
-        self.showInfo(args.show, butler, pipeline, registry, qgraph)
+        self.showInfo(args.show, pipeline, qgraph)
 
         if args.subcommand == "qgraph":
             # stop here
             return 0
 
         # execute
         if args.subcommand == "run":
+
```

Comment:
Yes, I was delaying looking at that post-preflight problem. Would I have to implement that myself here or is there something already in Registry that does that?

Reply:
There is nothing in Registry now, but there probably should be.  It might be the kind of thing that should work on a Subset-as-iterable-of-PreflightUnitRow.

Reply:
Well, at this point there is only a complete QuantumGraph (derived from Subset or read from from other source), so I'll have to walk that graph an do something (in a new ticket).